lib/node.js

```diff
@@ -524,16 +524,26 @@ declare module "readline" {
 }
 
 declare class stream$Stream extends events$EventEmitter {
-  pipe(dest: any, options: any): void;
+  pipe(dest: any, options?: any): void;
 }
 
 declare class stream$Readable extends stream$Stream {
   setEncoding(): void;
+  isPaused(): boolean;
+  pause(): void;
+  read(size?: number): ?(string | Buffer);
+  resume(): void;
+  unpipe(dest: any): void;
+  unshift(chunk: any): void;
+  wrap(oldReadable: any): stream$Readable;
 }
 
 declare class stream$Writable extends stream$Stream {
   close(): void;
+  end(): void;
   finish(): void;
+  setDefaultEncoding(): void;
+  write(): void;
```

Comment:
This has one required argument and 2 optional ones

https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback